### PR TITLE
7639 Remove old DRM+AGP code

### PR DESCRIPTION
**Pull request by gwr**
Let's move this along, shall we?
https://www.illumos.org/issues/7639

### CODE REVIEW

File: usr/src/man/man7i/Makefile [hunk 1]
```diff
@@ -41,7 +41,7 @@ _MANFILES=	audio.7i	\
 		visual_io.7i	\
 		vt.7i
 
-i386_MANFILES=	agpgart_io.7i
+i386_MANFILES=
```
[the remark this answers is not in the document]
Comment (jclulow) [reply usr/src/man/man7i/Makefile]: If you're going to put reviews up on Github it'd be best to reply using the Github facilities for review feedback and resolution management.

You wrote:
> I figured x86-specific man pages would likely come and go, so left it.

There's no empty `sparc_MANFILES` in here, so I'm still inclined to remove this one.  People copy-paste Makefiles perhaps more than anything else in the gate, and don't necessarily clip out the detritus.

File: usr/src/man/man7d/Makefile [hunk 2]
```diff
@@ -221,7 +220,6 @@ i386_MANFILES=	ahci.7d		\
 		nvme.7d		\
 		pcn.7d		\
 		qede.7d		\
-		radeon.7d	\
```
[the remark this answers is not in the document]
Comment (jclulow) [reply usr/src/man/man7d/Makefile]: >  Yes.  The radeon driver in illumos-gate is very old.  I don't think it
works with the current DRM driver and libraries.
There's a newer radeon driver in the later DRM code (in the gfx-drm
gate) but the common DRM driver is missing a feature it needs:
"translation table memory" (TTM).

> So the updated DRM code trades away ancient radeon support for a much
larger set of supported intel graphics h/w and a working para-virtual
driver for vmware.  That's a good trade.

OK, thanks.

File: usr/src/man/man7i/Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (gwr) [reply usr/src/man/man7i/Makefile]: removed

[the remark this answers is not in the document]
Comment (gwr) [reply usr/src/man/man7i/Makefile]: Oh, looks like I should remove this line too:
MANFILES=	$(_MANFILES) $($(MACH)_MANFILES)
Done (and push -f to this branch)

**Comment by gwr** [comment]
Updated list of reviewers

**Comment by gwr** [comment]
... and rebase

**Pull request merged**